cluster/gce/util.sh

```diff
@@ -1025,7 +1025,11 @@ EOF
 # cat the Kubelet config yaml for masters
 function print-master-kubelet-config {
   cat <<EOF
-enableDebuggingHandlers: false
+# Port-forwarding is needed to access the secure port of
+# kube-controller-manager and kube-scheduler
+# when retrieving metrics data
+# (see test/e2e/framework/metrics/metrics_grabber.go).
+enableDebuggingHandlers: true
```

Comment:
this should only be done in the test config, not the non-test config.

Reply:
Which "test config"? Do you mean some Prow job definition?

Reply:
I think it means `cluster/gce/config-test.sh`.
Take `KUBELET_TEST_LOG_LEVEL` as an exmaple, and use it here.

Reply:
So don't hard-code in `util.sh` but rather make it configurable in `config-test.sh`. I added `KUBELET_ENABLE_DEBUGGING_HANDLERS` (default true) and `MASTER_KUBELET_ENABLE_DEBUGGING_HANDLERS` (default `KUBELET_ENABLE_DEBUGGING_HANDLERS` and thus true).

This is still a change in how the cluster is created by default, though. If that is not desired, then we need to change Prow jobs.

Reply:
> This is still a change in how the cluster is created by default, though. If that is not desired, then we need to change Prow jobs.

Yes the default cluster without config-test.sh should not be enabling these without opt-in. CI should be using config-test?